### DISCUSSION TITLE
Post Quantum Key Exchange

### DISCUSSION THREAD

**Discussion by sheepymeh** [nginx-proxy/nginx-proxy#2564] [Ideas]
Are there any plans to enabling post quantum key exchange methods?

**Comment by buchdag** [comment]
Not as long as it will require compiling custom packages in the image to make it work.

see https://www.linode.com/docs/guides/post-quantum-encryption-nginx-ubuntu2404/

Edit: more generally speaking, nginx-proxy is intended to be compatible with the upstream nginx images when used in the two containers setup (nginx + docker-gen), so there is very little probability that we'll implement something that require significant deviation from those images.
